boards/arm/nrf5340_dk_nrf5340/doc/index.rst

```diff
@@ -0,0 +1,260 @@
+.. _nrf5340_dk_nrf5340:
+
+nRF5340-DK-NRF5340
+##################
+
+Overview
+********
+
+The nRF5340 DK NRF5340 is a single-board development kit for evaluation
+and development on the Nordic nRF5340 System-on-Chip (SoC).
+
+The nRF5340 is a dual-core SoC based on the Arm® Cortex®-M33 architecture, with:
+
+* a full-featured ARM Cortex-M33F core with DSP instructions, FPU, and
+  ARMv8-M Security Extension, running at up to 128 MHz, referred to as
+  the **Application MCU**
+* a secondary ARM Cortex-M33 core, with a reduced feature set, running at
+  a fixed 64 MHz, referred to as the **Network MCU**.
+
+The nrf5340_dk_nrf5340_cpuapp provides support for the Application MCU on
+nRF5340 SoC. The nrf5340_dn_nrf5340_cpunet provides support for the Network
+MCU on nRF5340 SoC.
+
+nRF5340 SoC provides support for the following devices:
+
+* :abbr:`ADC (Analog to Digital Converter)`
+* CLOCK
+* FLASH
+* :abbr:`GPIO (General Purpose Input Output)`
+* :abbr:`IDAU (Implementation Defined Attribution Unit)`
+* :abbr:`I2C (Inter-Integrated Circuit)`
+* :abbr:`MPU (Memory Protection Unit)`
+* :abbr:`NVIC (Nested Vectored Interrupt Controller)`
+* :abbr:`PWM (Pulse Width Modulation)`
+* RADIO (Bluetooth Low Energy and 802.15.4)
+* :abbr:`RTC (nRF RTC System Clock)`
+* Segger RTT (RTT Console)
+* :abbr:`SPI (Serial Peripheral Interface)`
+* :abbr:`UARTE (Universal asynchronous receiver-transmitter)`
+* :abbr:`USB (Universal Serial Bus)`
+* :abbr:`WDT (Watchdog Timer)`
+
+The `Nordic Semiconductor Documentation library`_
+contains the processor's information and the datasheet.
+
+Hardware
+********
+
+nRF5340 PDK has two external oscillators. The frequency of
```

Comment:
PDK or DK?

Reply:
Should be DK - that's a copy paste error, thanks.

Reply:
Will fix